src/app/components/course-card/CourseCardComponent.tsx

```diff
@@ -25,36 +27,51 @@ export const CourseCardComponent: React.FC<ICourseCardProps> = ({
   return (
     <Wrapper>
       <Title>
-        <div style={{ margin: "5px 0px 0px 20px" }}>{title}</div>
+        <TextContainer>{title}</TextContainer>
       </Title>
       <Content>
         <Image src={imageLink}></Image>
         <Description>
           <DescriptionText>{description}</DescriptionText>
         </Description>
-        <ParamsComponent>
-          <div style={{ margin: "5px 0px 0px 20px" }}>
-            <div>Lessons: {lessonsCount}</div>
-            <div style={{ marginTop: "10px" }}>
-              {skills ? (
-                <>
-                  Skills:
-                  {skills.map((skill, id) => (
-                    <li key={id}>{skill}</li>
-                  ))}{" "}
-                </>
-              ) : (
-                <>No skills</>
-              )}
-            </div>
-
-            <div style={{ marginTop: "10px" }}>Rating: {rating}</div>
-          </div>
-        </ParamsComponent>
+        <CardParamsComponent
+          lessonsCount={lessonsCount}
+          skills={skills}
+          rating={rating}
+        />
         <Button onClick={() => onClickFunction(id)}>
-          <div style={{ color: Colors.White }}>Choose</div>
+          <ButtonText>Choose</ButtonText>
         </Button>
       </Content>
     </Wrapper>
   );
 };
+
+export const CardParamsComponent: React.FC<ICardParamsComponentProps> = ({
+  lessonsCount,
+  skills,
+  rating,
+}) => {
+  const skillsList = () => {
```

Comment:
Краще винести цю функцію за межі компонента, а skills передавати у неї пропом.
Або взагалі замість функції зробити окремий компонент, щось типу такого:

```suggestion
  const SkillsList: FC<ISkillsListProps> = ({ skills }) => {
  if (!skills) return (<>No skills</>)
  
  return (
      <>
        Skills:
        {skills.map((skill, id) => (
          <li key={id}>{skill}</li>
        ))}
      </>
  )}
```

Reply:
Готово, винесла окремий загальний компонент.